src/normal/normal_generic.jl

```diff
@@ -60,7 +60,8 @@ entropy(N::AbstractNormal) = entropy(N)
 
 Computes the Kullback-Leibler divergence between N1 and N2.
 """
-kldivergence(N1::AbstractNormal, N2::AbstractNormal) = kldivergence(N1, N2)
+kldivergence(N1::AbstractNormal{T}, N2::AbstractNormal{T}) where {T<:Number} =
```

Comment:
Are there even `AbstractNormal{T}`s where `T` is not a `Number`? If not, you can simplify it again imo.

Reply:
So according to https://github.com/filtron/MarkovKernels.jl/blob/70c29bd75e250dbb079bba95e63b7819da0e5384/src/normal/normal.jl#L6 it's always a number, so no need to specialize?